pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -55,6 +57,8 @@ def forward(self, input_image: torch.Tensor) -> torch.Tensor:
 def style_transform(
     impl_params: bool = True, instance_norm: bool = True, edge_size: int = 256,
 ) -> transforms.Resize:
+    # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/train.lua#L152
+    # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/src/descriptor_net.lua#L17
```

Comment:
What `interpolation_mode` is given in the paper?

Reply:
I think that this was already a topic in a previous discussion. As the paper does not explicitly mention the mode or the way of resize. We have agreed on [image.scale](https://github.com/torch/image/blob/master/doc/simpletransform.md#res-imagescalesrc-width-height-mode) and the standard mode.